index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+    <html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <title>Fobadara's Portfolio</title>
+    </head>
+    <body>
+        <!-- If section or article is used here linter error displays missing head tag hence the use of div -->
+        <div class="main-container">
```

Comment:
- [ ] As requested by the previous reviewer, kindly make sure that your design matches the provided [Figma design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) as much as possible. Kindly ensure that your design matches the provided colors, for background, alignments: margins, and paddings.

Your design:
![image](https://user-images.githubusercontent.com/12745474/125703357-ed1ddc12-ba9a-4824-b493-15286072118b.png)

The original design:
![image](https://user-images.githubusercontent.com/12745474/125703477-bc2e2a2a-b991-45f6-9c68-684ff1153ced.png)

Reply:
- [ ] It looks like you change the background color as requested, but you forgot the margins and paddings, if you observe the original, this has some spaces between the end screen and the info in `<nav>`, according to an image that's the request for my last partner. I'm pretty sure you can improve this a little bit more. 🐶 
![image](https://user-images.githubusercontent.com/20563252/125781194-e747f580-68a3-4ea0-9c3c-8320efe011cd.png)

Reply:
- [ ] It looks like you change the background color as requested, but you forgot the margins and paddings, if you observe the original, this has some spaces between the end screen and the info in `<nav>`, according to an image that's the request for my last partner. I'm pretty sure you can improve this a little bit more. 🐶 
![image](https://user-images.githubusercontent.com/20563252/125781194-e747f580-68a3-4ea0-9c3c-8320efe011cd.png)